test/e2e/core/mapContextMenu.test.js

```diff
@@ -534,4 +538,31 @@ test.describe("Playwright Map Context Menu Tests", () => {
     );
     expect(layerCheck).toEqual(false);
   });
+
+  test("Checking Context Menu Items Names In Order", async () => {
+    let back = await page.$eval("div > div.mapml-contextmenu > button:nth-child(1)",(btn) => btn.textContent);
+    expect(back).toEqual('Back (B)');
+    let forward = await page.$eval("div > div.mapml-contextmenu > button:nth-child(2)",(btn) => btn.textContent);
+    expect(forward).toEqual('Forward (F)');
+    let reload = await page.$eval("div > div.mapml-contextmenu > button:nth-child(3)",(btn) => btn.textContent);
+    expect(reload).toEqual('Reload (R)');
+    let fullScreen = await page.$eval("div > div.mapml-contextmenu > button:nth-child(4)",(btn) => btn.textContent);
+    expect(fullScreen).toEqual('View fullscreen (E)');
+    let copy = await page.$eval("div > div.mapml-contextmenu > button:nth-child(6)",(btn) => btn.textContent);
+    expect(copy).toEqual('Copy (C)');
+    let paste = await page.$eval("div > div.mapml-contextmenu > button:nth-child(8)",(btn) => btn.textContent);
+    expect(paste).toEqual('Paste (P)');
+    let controls = await page.$eval("div > div.mapml-contextmenu > button:nth-child(10)",(btn) => btn.textContent);
+    expect(controls).toEqual('Toggle Controls (T)');
+    let debug = await page.$eval("div > div.mapml-contextmenu > button:nth-child(11)",(btn) => btn.textContent);
+    expect(debug).toEqual('Toggle Debug Mode (D)');
+    let source = await page.$eval("div > div.mapml-contextmenu > button:nth-child(12)",(btn) => btn.textContent);
```

Comment:
Good work testing this, this test is very important! I wonder if you should also add test to check `button:nth-child(13)` to make sure that does not exist, will help with testing if we decide to add more items to the contextmenu.